claims_common.go

```diff
@@ -0,0 +1,339 @@
+// Copyright 2021 Contributors to the Veraison project.
```

Comment:
Question: Does Copyright script (from Sergei) run here?

Reply:
not unless someone runs it

Reply:
done in ce5ece9